os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -296,6 +296,28 @@ static int assert_tracecallback(FAR struct usbtrace_s *trace, FAR void *arg)
 }
 #endif
 
+/****************************************************************************
+ * Name: up_show_asserted_tcb
+ ****************************************************************************/
+
+void up_show_asserted_tcb(struct tcb_s *tcb)
```

Comment:
This function shows tcb information selected. In the function operation of view, there is no asserted content. Let's rename it as `up_show_tcb_info`

Reply:
If we rename it, then later we could use it to print all of alive tcb.